.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!---
+Please read our [Contributing Guidelines](https://github.com/harvesthq/chosen/blob/master/contributing.md) before opening this bug report or feature request.
+
+Following these guidelines will help us get back to you more quickly, and will show that you care about making Chosen better just like we do!
+-->
+
+## Bug Report Template
+
+Describe your issue here.
+
+### My environment
+A summary of the issue and the browser/OS environment in which it occurs.
+
+### Steps to reproduce
+Tell us how to reproduce this issue.
+
+1. This is the first step
+2. This is the second step
+3. Further steps, etc.
+
+Please provide a working demo, you can use [this template](https://jsfiddle.net/hyktf3he) as a base.
+
+### Expected behaviour
```

Comment:
@Pezmc I think you mean "behavior".

:us: :us: :us: :us: :us: :us: :us: :us: :us: